Airports/EGPH/Sids.txt

```diff
@@ -1,6 +1,6 @@
-SID:EGPH:06:GOSAM1D:D068A D051C GOW35 GOW30 CUMBO GOSAM
-SID:EGPH:24:GOSAM1C:UW D262E D262J MAVIX GOSAM
-SID:EGPH:06:GRICE4D:D068A D051C SAB46 GRICE
+SID:EGPH:06:GOSAM1D:D065A D050C GOW35 GOW30 CUMBO GOSAM
+SID:EGPH:24:GOSAM1C:UW D081E D081J MAVIX GOSAM
```

Comment:
Looks like a reciprocal of the actual track here?

Reply:
Bit confused what you mean with this - to me the track looks okay, and the extra waypoints are from simbrief to match what aircraft will have in their system (i.e. still direct from UW -> MAVIX, just with what they'll have in the computer). If its a data issue then perhaps @hazzas-99 will know because its from his simbrief.